src/orange/transform/Translation.hh

```diff
@@ -0,0 +1,127 @@
+//----------------------------------*-C++-*----------------------------------//
+// Copyright 2021-2023 UT-Battelle, LLC, and other Celeritas developers.
+// See the top-level COPYRIGHT file for details.
+// SPDX-License-Identifier: (Apache-2.0 OR MIT)
+//---------------------------------------------------------------------------//
+//! \file orange/Translator.hh
+//---------------------------------------------------------------------------//
+#pragma once
+
+#include "corecel/cont/Span.hh"
+#include "corecel/math/Algorithms.hh"
+#include "corecel/math/ArrayOperators.hh"
+#include "orange/OrangeTypes.hh"
+
+namespace celeritas
+{
+//---------------------------------------------------------------------------//
+/*!
+ * Storage implementation for transformations with rotation.
+ *
+ * The input translation and rotation are specified by rotating the daughter's
```

Comment:
Rephrase to remove rotation

Reply:
This and the above comment were a copy-paste error from Transformation. Thanks for catching it!